rust/windows-client/src-tauri/src/client/gui.rs

```diff
@@ -63,6 +63,28 @@ pub(crate) fn run(params: client::GuiParams) -> Result<()> {
 
     let advanced_settings = settings::load_advanced_settings().unwrap_or_default();
 
+    // If the log filter is unparsable, show an error and use the default
+    // Fixes <https://github.com/firezone/firezone/issues/3452>
+    let advanced_settings =
+        match tracing_subscriber::EnvFilter::from_str(&advanced_settings.log_filter) {
+            Ok(_) => advanced_settings,
+            Err(_) => {
```

Comment:
We could also write a `warn` to the log that contains the error so that an advanced user could fix it.